app/assets/stylesheets/admin/angular.css.scss

```diff
@@ -0,0 +1,4 @@
+// https://docs.angularjs.org/api/ng/directive/ngCloak
+[ng\:cloak], [ng-cloak], [data-ng-cloak], [x-ng-cloak], .ng-cloak, .x-ng-cloak {
+  display: none !important;
+}
```

Comment:
this is not pretty but really nice that we are hiding the "no order found" message!
do we need to keep this file name and location? it would be better to name the file angula-ng-cloak.css.scss and move it under /components. because it has the !important I think this will also work, right?

Reply:
I just replicated what we have in Darkswarm. But I agree that it's better in components. I will move it.